tests/bluetooth/host/buf/bt_buf_get_evt/kconfig_override.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Demant
```

Comment:
```suggestion
 * Copyright (c) 2022 Nordic Semiconductor ASA
```

Reply:
Hi @thoh-ot , Could you review again? That change request has been applied long time ago.

Reply:
@thoh-ot Could you review again? Your change request is blocking the PR